addons/website_sale_comparison/views/snippets.xml

```diff
@@ -14,6 +14,12 @@
                    data-no-preview="true"
                    data-reload="/"/>
     </xpath>
+    <xpath expr="//we-button[hasclass('o_we_buy_now_btn')]" position="after">
+        <we-button title="Compare" class="fa fa-fw fa-exchange"
+                   data-customize-website-views="website_sale_comparison.add_to_compare"
+                   data-no-preview="true"
+                   data-reload="/"/>
+    </xpath>
```

Comment:
@paju-odoo do you think a snippet to customize (hide/show) the `Compare` button from the product page is useful ?
There is a snippet to enable/disable the comparison feature in the snippets of the `/shop` page, but should we allow enabling/disabling the whole feature from the product page as well ?

Reply:
@paju-odoo we can also make it specific to this page if you prefer

Reply:
She's on leave right now, but she'll probably take a look early next week ;)

Reply:
@paju-odoo  thanks for the feedback
I've applied your suggestion